src/components/EmailNotifications/DeleteEmail.js

```diff
@@ -0,0 +1,101 @@
+import React, { useCallback } from 'react'
+import { Button, GU, textStyle, useTheme } from '@aragon/ui'
+
+import emailNotifcationIllustration from '../../assets/emailNotifications.svg'
+
+const DeleteEmail = React.memo(function DeleteEmail({
+  isModal,
```

Comment:
Given the amount of things changing, and that this is not a really complex component, what do you think about just making two different components?

Reply:
Not sure about this we are setting the same header, same text,  not sure why we should repeat that.
And about the mode could be, i just prefer the boolean but can change it if needed

Reply:
for the isModal same answer than here https://github.com/aragon/court-dashboard/pull/306#discussion_r423143611

Reply:
Should we use `Inside` and `useInside()`? `Modal` in aragonUI doesn’t declare an `Inside` scope, but we could do it ourselves anyway:

```jsx
<Modal>
  <Inside name="Modal">
    // …
  </Inside>
</Modal>
```

```jsx
function DeleteEmail() {
  const [insideModal] = useInside('Modal')
  // …
}
```

Both `Inside` and `useInside` are exported by aragonUI: https://github.com/aragon/use-inside

Reply:
Oh interesting, yes, I hadn't considered this!

Reply:
And i never saw this comment!!!